### PR TITLE
Remove old SSP CRDs

### PR DESCRIPTION
**Pull request by nunnatsa**
**Release note**:

```release-note
Remove old, unused CRDs from SSP
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by nunnatsa** [comment]
/hold

Waiting for a bug fix from SSP

File: pkg/controller/hyperconverged/hyperconverged_controller.go [hunk 1]
```diff
@@ -1634,7 +1674,27 @@ func (r *ReconcileHyperConverged) setLabels(req *hcoRequest) {
...
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			req.logger.Error(err, fmt.Sprintf("failed to read the %s CRD; %s", crdName, err.Error()))
+		}
+	} else {
+		err = r.client.Delete(req.ctx, found)
+		if err != nil {
+			req.logger.Error(err, fmt.Sprintf("failed to remove the %s CRD; %s", crdName, err.Error()))
+		}
+	}
```
Comment (tiraboschi) [inline pkg/controller/hyperconverged/hyperconverged_controller.go:1697]: I think it's worth to log at info level that we successfully deleted CRD X

**Comment by nunnatsa** [comment]
Can we override continuous-integration/travis-ci/pr ? The test passed but travis failed to report status:

https://travis-ci.com/github/kubevirt/hyperconverged-cluster-operator/builds/173642736

Only option to solve it is to run **everything** again

**Comment by nunnatsa** [comment]
travis failed to report the status again. And again - it passed: 
https://travis-ci.com/github/kubevirt/hyperconverged-cluster-operator/builds/173648242

**Comment by tiraboschi** [comment]
Now it really fails on `make sanity`:
```
diff --git a/cmd/hyperconverged-cluster-operator/main.go b/cmd/hyperconverged-cluster-operator/main.go
index d24ef7b..9e2b985 100644
--- a/cmd/hyperconverged-cluster-operator/main.go
+++ b/cmd/hyperconverged-cluster-operator/main.go
@@ -15,9 +15,9 @@ import (
 	"github.com/spf13/pflag"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/rest"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 	"os"
 	"runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
make: *** [sanity] Error 1
The command "make sanity" exited with 2.
```

**Pull request merged**